pkg/controllers/qat/controller.go

```diff
@@ -89,7 +91,16 @@ func (c *controller) NewDaemonSet(rawObj client.Object) *apps.DaemonSet {
 
 	annotations := devicePlugin.ObjectMeta.DeepCopy().Annotations
 
+	if Unconfined {
```

Comment:
We added `annotations := devicePlugin.ObjectMeta.DeepCopy().Annotations` to address this Apparmor case and it's documented here https://github.com/intel/intel-device-plugins-for-kubernetes/blob/37b40af973370d47d71d9fafeb3670cbfc1bc7ff/deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml#L5-L10. 

Perhaps we can make it so that `deployments/qat_plugin/base/intel-qat-plugin.yaml` adds the annotation by default and the CRD annotation can be used to override the setting if it's needed.